README.md

```diff
@@ -154,8 +154,18 @@ level of the project, if it exists.
 
   * The file is expected to conform to the [RPM Changelog][RPM CHANGELOG]
     format described in the Fedora packaging guidelines
+
   * The file MUST start with a well-formatted RPM changelog string, or it will
     be ignored.
+
+    Example:
+
+        * Mon Nov 06 2017 Tom Smith <tom.smith@simp.com> - 3.8.0-0
+        - Add feature x
+
+    **Important:** Note the leading zero in "`Nov 05`".  It is a convention
+
+    that is **required** by our CHANGELOG validation tasks.
```

Comment:
it is a convention required by rpmbuild

Reply:
@lnemsick-simp  Can you demonstrate that?

I wasn't sure before writing this patch, so I [tested `rpmbuild -ba` with a small .spec file](https://gist.github.com/op-ct/e675ebb652b0625316a73b7044b192b4).  The RPM built successfully on EL6 and EL7, and running `rpm -qp FILE --changelog` provided a correct result.

There are also [numerous][1] [examples][2] [of 1-digit days][3] in our modules' historical CHANGELOGs.

[1]: https://github.com/simp/pupmod-simp-ssh/blob/6.2.1/CHANGELOG#L31
[2]: https://github.com/simp/pupmod-simp-cron/blob/0.0.2/CHANGELOG#L7
[3]: https://github.com/simp/pupmod-simp-gnome/blob/6.0.2/CHANGELOG#L27